README.md

```diff
@@ -484,35 +484,35 @@ HubProxy can be configured using either command-line flags or a YAML configurati
 
 Sensitive configuration values should be provided through environment variables:
 
-- `GITHUB_WEBHOOK_SECRET`: GitHub webhook secret for verification (required)
+- `HUBPROXY_WEBHOOK_SECRET`: GitHub webhook secret for verification (required)
 
 ### Configuration File
 
 Create a `config.yaml` file (see `config.example.yaml` for a template) with your desired settings.
 
 ```yaml
 # Target URL to forward webhooks to
-target_url: "http://your-service:8080/webhook"
+target-url: "http://your-service:8080/webhook"
 
 # Log level (debug, info, warn, error)
-log_level: info
+log-level: info
 
 # Validate that requests come from GitHub IPs
-validate_ip: true
+validate-ip: true
 
 # Tailscale configuration (optional)
-ts_authkey: ""
-ts_hostname: hubproxy
+ts-authkey: ""
+ts-hostname: hubproxy
 
 # Database configuration
-db_type: sqlite  # sqlite, mysql, or postgres
-db_dsn: hubproxy.db
+db-type: sqlite  # sqlite, mysql, or postgres
+db-dsn: hubproxy.db
```

Comment:
Would you rather support URI style, `--db sqlite3://hubproxy.db` instead?

Reply:
Yeah absolutely. `dn-dsn` is unclear and we're repeating ourselves.